src/app_service/service/message/service.nim

```diff
@@ -713,22 +713,7 @@ proc getRenderedText*(self: Service, parsedTextArray: seq[ParsedText]): string =
           result = result & self.renderInline(child)
         result = result & "</p>"
       of PARSED_TEXT_TYPE_BLOCKQUOTE:
-        var
-          blockquote = escape_html(parsedText.literal)
-          lines = toSeq(blockquote.split(NEW_LINE))
-        for i in 0..(lines.len - 1):
-          if i + 1 >= lines.len:
-            continue
-          if lines[i + 1] != "":
-            lines[i] = lines[i] & "<br/>"
-        blockquote = lines.join("")
-        result = result & fmt(
-          "<table class=\"blockquote\">" &
-            "<tr>" &
-              "<td class=\"quoteline\" valign=\"middle\"></td>" &
-              "<td>{blockquote}</td>" &
-            "</tr>" &
-          "</table>")
+        result = result & "blockquote:" & escape_html(parsedText.literal)
```

Comment:
I meant wrapping it in literally the HTML `<blockquote>foo</blockquote>` tag :) As @alexjba pointed out, it will now trigger on any text containing the string "blockquote:"

Reply:
https://www.w3schools.com/tags/tag_blockquote.asp

Reply:
ooh yeah, I tried that yesterday but can't remember now why dropped it! so I've added <blockquote> and QML now checks startsWith <blockquote> and endsWith </blockquote> which should be fine I guess.